draft-pda-protocol.md

```diff
@@ -73,7 +73,16 @@ normative:
 
 --- abstract
 
-TODO: writeme
+There are many situations in which it is desirable to take
+measurements of data which people consider sensitive.  In these cases,
+the entity taking the measurement is usually not interested in
+people's individual responses but rather in aggregated data but
+conventional methods require collecting individual responses and then
+aggregating them, thus representing a threat to user privacy and
+rendering many such measurements difficult and impractical.
+This document describes a privacy preserving measurement (PPM)
```

Comment:
```suggestion
This document describes a multi-party privacy preserving measurement (PPM)
```
Take it or leave it, but IMO clarifying that this involves multiple entities seems like a useful thing to be up front about.